server/src/services/series.js

```diff
@@ -37,6 +45,97 @@ const aggregateSeries = async (seriesIds) => {
     ]);
 };
 
+const filterSeries = async ({ name, status, genre }, pageNumber, pageLimit) => {
+    const aggregationQuery = [];
+
+    name && aggregationQuery.push({ $addFields: { searchIndex: { $indexOfCP: ["$name", String(name)] } } }, { $match: { searchIndex: { $ne: -1 } } });
+    status && aggregationQuery.push({ $match: { status: { $eq: status } } });
```

Comment:
`status: { $in: status }`
you can get multiple statuses from checkboxes, a user that marks "Ended" and "Canceled" will want to find all canceled/ended series. Therefore `$in` suits here